server/auth/auth.go

```diff
@@ -660,6 +668,23 @@ func (auth *AuthManager) Penalize(user account.AccountID, rule account.Rule) err
 	return nil
 }
 
+// Unban forgives a user, allowing them to resume trading.
+func (auth *AuthManager) Unban(user account.AccountID) error {
+	client := auth.user(user)
+	if client != nil {
+		client.resume()
+	}
+
+	if err := auth.storage.OpenAccount(user /*client.acct.ID*/); err != nil {
+		return err
+	}
+
+	log.Debugf("user %v unbanned", user)
+
+	// TODO: notify client of account status change?
```

Comment:
Maybe at some point.  It's not a normal event and the user can always retry.  Notifying of the ban is the more important part of course.